src/content/docs/iast/troubleshooting/unable-to-detect-vulnerabilities-for-golang-application.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: No vulnerabilities detected for Golang application
+tags:
+  - IAST
+  - New Relic IAST
+  - Troubleshooting
+metaDescription: Troubleshoot Golang application if no vulnerabilities are detected.
+freshnessValidatedDate: 2024-07-16
+---
+
+## Problem
+
+You don't see any vulnerability for your Golang application.
+
+## Solution
+
+- For Golang, ensure that you imported the required instrumentation packages of libraries and frameworks.
+
+    For example, your application is using libraries for [Mongo DB](http://go.mongodb.org/mongo-driver/mongo). For this specific library, you need to import this [instrumentation package](https://github.com/newrelic/csec-go-agent/tree/main/instrumentation/csec_mongodb_mongo) from newrelic:
```

Comment:
"...Mongo DB, then you must import the instrumentation package library..."

Check capitlaizaton for all proper nouns